### PR TITLE
websocket does not establish under 'server' mode

### LINKED ISSUE #312: websocket does not establish under 'server' mode.
#### Issue description
websocket **does not** establish under 'server' mode.

```js
// yes, 'server' is default value of 'analyzerMode'
new BundleAnalyzerPlugin({analyzerMode: 'server'})
```
> I am using `webpack-bundle-analyzer` as  webpack plugin.

I have researched. This issue is related PR #215.

#### Testing
In below code, you can find that `reporter.html` uses `enableWebSocket` before setting.

### viewer.jsx
```js
console.log('use enableWebSocket', window.enableWebSocket);
// Initializing WebSocket for live treemap updates
let ws;
try {
  if (window.enableWebSocket) {
    ws = new WebSocket(`ws://${location.host}`);
  }
} catch (err) {
  console.warn(
    "Couldn't connect to analyzer websocket server so you'll have to reload page manually to see updates in the treemap"
  );
}
```

### viewer.ejs
```html
  <%- include('script', { filename: 'viewer.js' }) %>
  </head>
  <body>
    <div id="app"></div>
    <script>
      window.chartData = <%- escapeJson(chartData) %>;
      window.defaultSizes = <%- escapeJson(defaultSizes) %>;
      window.enableWebSocket = <%- escapeJson(enableWebSocket) %>;
      console.log('set Websocket', window.enableWebSocket);
    </script>
```

Test result in browser.

image

**I am making a PR to fix it.**

#### Technical info
* Webpack Bundle Analyzer version: 3.5.0
* Webpack version: 4.39.3
* Node.js version: 10.15.3
* npm/yarn version: 6.11.3
* OS: MacOS

### PR DESCRIPTION
**Pull request by gaokun**
fix #312 .

set `enableWebSocket` before using.

### CODE REVIEW

File: views/viewer.ejs [hunk 1]
```diff
@@ -6,15 +6,17 @@
...
+      window.enableWebSocket = <%- escapeJson(enableWebSocket) %>;
+    </script>
     <%- include('script', { filename: 'viewer.js' }) %>
   </head>
 
   <body>
     <div id="app"></div>
     <script>
       window.chartData = <%- escapeJson(chartData) %>;
-      window.defaultSizes = <%- escapeJson(defaultSizes) %>;
```
Comment (valscion) [inline views/viewer.ejs]: Can you keep this `window.defaultSizes` as it was, as it seems to be unrelated to your fix?

Comment (gaokun) [reply views/viewer.ejs]: sure, no problem.
> I thought `defaultSizes` is a quite simple variable, so I moved it up.

**Comment by valscion** [comment]
Thanks :+1:
